tests/utils/helpers.py

```diff
@@ -0,0 +1,46 @@
+from collections import OrderedDict, namedtuple
+from tests.mocks.targets import CreditFraud
+from tests.mocks.targets import Digits
+from tests.mocks.targets import DigitKeras
+from tests.mocks.targets import SatelliteImages
+
+def get_target_class(target: str):
+    """loads attack-target combinations from json as dictionary"""
+    target_classes = {
+        "CreditFraud": CreditFraud(),
+        "Digits": Digits(),
+        "DigitKeras": DigitKeras(),
+        "SatelliteImages": SatelliteImages()
+    }
+    return target_classes[target]
+
+def extract_test_data(key: str, load_module_test_data):
+    """gets only required portion from module test data dictionary as test object"""
+    if key not in load_module_test_data:
+        return
```

Comment:
I was thinking we should throw an exception when key not found in `load_module_test_data` something like `raise KeyError(f'{key} not found in the modules{load_module_test_data}. Please provide a valid key.')`?